src/pages/home/report/ReportActionsView.js

```diff
@@ -335,15 +340,14 @@ class ReportActionsView extends React.Component {
         // <InvertedFlatList /> are implemented on native and web/desktop which leads to
         // the unread indicator on native to render below the message instead of above it.
             <View>
-                {this.unreadActionCount > 0 && index === this.unreadActionCount - 1 && (
-                    <UnreadActionIndicator animatedOpacity={this.unreadIndicatorOpacity} />
-                )}
                 <ReportActionItem
                     reportID={this.props.reportID}
                     action={item.action}
                     displayAsGroup={this.isConsecutiveActionMadeByPreviousActor(index)}
                     onLayout={onLayout}
                     needsLayoutCalculation={needsLayoutCalculation}
+                    onMarkAsUnread={() => this.onMarkAsUnread(item.action.sequenceNumber - 1)}
```

Comment:
It will be better for performance to pass the `sequenceNumber` from `ReportActionItem` and just pass `this.onMarkAsUnread` directly like:

```jsx
onMarkAsUnread={this.onMarkAsUnread}
```

as far as React is concerned seeing `() => someMethod()` means it gets interpreted as a "new function" on every render. This is usually OK but probably should be avoided in our most performance sensitive components like this one.